tests/lib/ocpp/v201/test_charge_point.cpp

```diff
@@ -175,6 +181,27 @@ class ChargePointFixture : public DatabaseTestingUtils {
     std::unique_ptr<TestChargePoint> charge_point = create_charge_point();
     boost::uuids::random_generator uuid_generator = boost::uuids::random_generator();
 
+    std::shared_ptr<DatabaseHandler> create_database_handler() {
```

Comment:
what is the difference between the database handler and the device model? Why are we not using an in-memory structure for the database as well?

Reply:
The device model is a specific, separate database for the device model concept within OCPP. It contains configuration values for the EVSEs, ChargingStation, etc. The database handler is for the core database of libocpp, which houses the data for charging profiles, transactions, authorization, and other persistent data.

The ChargePoint expects us to have a database file in a specific spot, `core_database_path/cp.db`. You can see this [here](https://github.com/EVerest/libocpp/blob/main/lib/ocpp/v201/charge_point.cpp#L121). Because of this, we set up our test database handler the same way. We're doing this for now for consistency, and we hope to change this if the ChargePoint is changed to allow for in-memory databases.